packages/components/nodes/chatmodels/ChatGoogleGenerativeAI/ChatGoogleGenerativeAI.ts

```diff
@@ -74,6 +76,72 @@ class GoogleGenerativeAI_ChatModels implements INode {
                 step: 0.1,
                 optional: true,
                 additionalParams: true
+            },
+            {
+                label: 'topK',
+                name: 'topK',
+                type: 'number',
+                step: 0.1,
+                optional: true,
+                additionalParams: true
+            },
+            {
+                label: 'Harm Category',
+                name: 'harmCategory',
+                type: 'multiOptions',
+                description:
+                    'Refer to <a target="_blank" href="https://cloud.google.com/vertex-ai/docs/generative-ai/multimodal/configure-safety-attributes#gemini-TASK-samples-go">official guide</a> on how to use Harm Category',
+                options: [
+                    {
+                        label: 'Dangerous',
+                        name: HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT
+                    },
+                    {
+                        label: 'Harassment',
+                        name: HarmCategory.HARM_CATEGORY_HARASSMENT
+                    },
+                    {
+                        label: 'Hate Speech',
+                        name: HarmCategory.HARM_CATEGORY_HATE_SPEECH
+                    },
+                    {
+                        label: 'Sexually Explicit',
+                        name: HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT
+                    }
+                ],
+                optional: true,
+                additionalParams: true
+            },
+            {
+                label: 'Harm Block Threshold',
+                name: 'harmBlockThreshold',
+                type: 'multiOptions',
+                description:
+                    'Refer to <a target="_blank" href="https://cloud.google.com/vertex-ai/docs/generative-ai/multimodal/configure-safety-attributes#gemini-TASK-samples-go">official guide</a> on how to use Harm Block Threshold',
+                options: [
+                    {
+                        label: 'Low and Above',
+                        name: HarmBlockThreshold.BLOCK_LOW_AND_ABOVE
+                    },
+                    {
+                        label: 'Medium and Above',
+                        name: HarmBlockThreshold.BLOCK_MEDIUM_AND_ABOVE
+                    },
+                    {
+                        label: 'None',
+                        name: HarmBlockThreshold.BLOCK_NONE
+                    },
+                    {
+                        label: 'Only High',
+                        name: HarmBlockThreshold.BLOCK_ONLY_HIGH
+                    },
+                    {
+                        label: 'Threshold Unspecified',
+                        name: HarmBlockThreshold.HARM_BLOCK_THRESHOLD_UNSPECIFIED
+                    }
+                ],
+                optional: true,
+                additionalParams: true
```

Comment:
default: BLOCK_MEDIUM_AND_ABOVE

Reply:
The default no need to be set here if they did not choose any value from `harmCategory`.